entropylab/api/plot.py

```diff
@@ -99,8 +99,11 @@ def __init__(self) -> None:
         super().__init__()
 
     def plot_plotly(self, figure: go.Figure, data, **kwargs) -> None:
-        # TODO: Validate type of data
-        return px.imshow(data)
+        fig = px.imshow(data)
```

Comment:
`fig` and `figure` are a bit confusing. Is `fig` of type `Figure`? Maybe rename `fig` to `imshow_figure`?